styles/variables-silicon.less

```diff
@@ -21,7 +21,9 @@
 
 // Checkbox
 // ---------------------------------------
-@agate-checkbox-border-radius: 6px;
+@agate-checkbox-size: 30px;
+@agate-checkbox-border: 3px solid transparent;
```

Comment:
We already defined the border color in `@agate-checkbox-border-color: transparent;`. In addition any colors should not be defined in the variable less files. Why don't you just define a `border-width` because the default for a `border-style is a `solid`?

```
.Checkbox_toggleIcon__3FAw8.silicon {
    width: 1.25rem;
    height: 1.25rem;
    border: 0.125rem solid transparent; // This border color will be overwritten by the following border-color.
    border-color: transparent;
    border-radius: 0;
    color: #ffffff;
    background-color: #f1304f;
}
```